distro/README.md

```diff
@@ -52,19 +56,36 @@ cd release-from-archive
 tar xzf ../riff-release.tar.gz
 ```
 
-Relocate the images in the release into the `relocated` directory, specifying a new `dev.local` registry and `u` user, and rewriting all the release yaml with the relocated image names/tags. Using `--flatten` is required for registries which don't support deep hierarchies of image names.
+Relocate the images into a `relocated` directory, specifying the `registry` and `user-repo`. This will create a new manifest and new release yaml files, prefixing all the image names with `registry`/`user-repo`.
+
 ```sh
-riff image relocate -m manifest.yaml -r dev.local -u u -i image-manifest.yaml -o relocated --flatten
+export REGISTRY_NAME=docker.io  # replace with your private registry name
```

Comment:
Should that be named `REGISTRY_HOST` rather?

Reply:
Yes, it's a host name. Well, not quite, as it could include a port.